packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleVideoViewer/SingleVideoViewerContainer.js

```diff
@@ -74,7 +149,32 @@ class SingleVideoViewerContainer extends React.Component {
     //   return null
     // }
 
-    return <SingleVideoViewer url={vid}></SingleVideoViewer>
+    return (
+      <div>
+        <div>
+          <SingleVideoViewer
+            playerRef={this.handlePlayerRef}
+            url={vid}
+            isPlaying={isPlaying}
+            playbackRate={playbackRate}
+            onProgress={this.handleVideoProgress}
+            onDuration={this.handleVideoDuration}
+          ></SingleVideoViewer>
```

Comment:
If `SingleVideoViewer` doesn't have children, then this can be a single closed tag.